src/main.ts

```diff
@@ -56,6 +56,8 @@ async function bootstrap() {
     .setDescription(SWAGGER_CONSTANTS.DESCRIPTION)
     .setVersion(SWAGGER_CONSTANTS.VERSION)
     .addTag(SWAGGER_TAGS.DEFAULT) // Add a tag for API grouping
+    .addTag(SWAGGER_TAGS.SURVEYCONFIG)
+    .addTag(SWAGGER_TAGS.SURVEYPARAMETER)
```

Comment:
Please remove this piece of code. Not Required.

Reply:
removed